be/src/runtime/stream_load/stream_load_context.h

```diff
@@ -113,7 +113,9 @@ class StreamLoadContext {
     bool unref() { return _refs.fetch_sub(1) == 1; }
 
 public:
-    MemTracker* mem_tracker = nullptr;
+    std::shared_ptr<RuntimeProfile> runtime_profile;
+    std::shared_ptr<MemTracker> query_mem_tracker;
```

Comment:
If these two variable is not used, better not to copy it.

Reply:
query_mem_tracker is the parent of instance_mem_tracker
runtime_profile will be used by [consumption] of mem_tracker to record peak memory

Reply:
I see, please comment this in the code.

Reply:
Fixed